libeos-updater-util/flatpak.c

```diff
@@ -242,13 +251,14 @@ parse_flatpak_ref_from_entry (JsonObject      *entry,
     return FALSE;
 
   *out_name = name;
+  *out_branch = branch;
   *out_ref_kind = kind;
 
   return TRUE;
 }
 
-static const gchar *
-eos_updater_get_system_architecture_string (void)
+const gchar *
+euu_get_system_architecture_string (void)
```

Comment:
Nitpick: Please can we have gtk-doc comments for all public API? Could be done as a follow-up PR. Here, for example, it would mention the envvar, and give some examples of the format the architecture is returned in.

Reply:
(Done in #155.)